apps/theme/layouts/forms/DefaultFormLayout/fields/components/FieldLabel.tsx

```diff
@@ -1,11 +1,35 @@
+import * as React from "react";
 import styled from "@emotion/styled";
+import { ReactComponent as Asterisk } from "@material-symbols/svg-400/outlined/emergency.svg";
 import theme from "../../../../../theme";
 
-export const FieldLabel = styled.label`
+export const FieldLabelStyled = styled.label`
     width: 100%;
     display: inline-block;
     margin: 0 0 5px 1px;
+
     ${theme.breakpoints.mobile} {
         text-align: left !important;
     }
+
+    svg {
+        width: 10px;
+        height: 10px;
+        fill var(--mdc-theme-primary, #fa5723);
```

Comment:
We don't want to use the SCSS vars anymore.

For all theme-related values, please rely on the `theme.ts` file. In this file, there's an example of accessing theme-defined breakpoints (`theme.breakpoints.mobile`).